src/installer.ts

```diff
@@ -103,13 +132,13 @@ export async function getNode(
 async function getInfoFromManifest(
   versionSpec: string,
   stable: boolean,
-  token: string
+  token: string | undefined
 ): Promise<INodeVersionInfo | null> {
   let info: INodeVersionInfo | null = null;
   const releases = await tc.getManifestFromRepo(
     'actions',
     'node-versions',
-    token
+    token || ''
```

Comment:
Need to modify tc.getManifestFromRepo to allow undefined/empty token.

Today it assumes token is not empty and always adds authorization header.

For E2E testing, i hacked it locally

Reply:
OK.  Update it and I'll release today.

Reply:
Here's the PR https://github.com/actions/toolkit/pull/460

Reply:
The toolkit is merged and released.  Do you want to update this to pass `token: ${token}`?